pynicotine/plugins/plugin_debugger/__init__.py

```diff
@@ -26,114 +28,131 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
+        self.settings = {}
+        self.metasettings = {}
+
+        for method_name, method in getmembers(self, ismethod):
+            if method.__module__ != __name__ or method_name.startswith("_"):
+                # Convenience function from BasePlugin, don't include these
+                continue
```

Comment:
`Plugin.__dict__` should give you only the members of the subclass. As a bonus, it will keep the order the methods are defined.